app/Http/Controllers/AttendanceController.php

```diff
@@ -53,18 +53,18 @@ public function store(Request $request)
         unset($request['includeName']);
 
         // Variables for comparison below
-        $today = date('Y-m-d');
+        $date = $request->has('created_at') ? $request->input('created_at') : date('Y-m-d');
```

Comment:
This can be simplified to `$date = $request->input('created_at', date('Y-m-d');`